### ISSUE TITLE
train with own data/objects

### ISSUE THREAD

**Issue by lupotto** [eriklindernoren/PyTorch-YOLOv3#16]
Hi Erik,

First of all, thanks for the repo, is really clean and helpful.

I would like to ask you if is it possible to train with your own classes, similar to AlexeyAB implementation (https://github.com/AlexeyAB/darknet#how-to-train-to-detect-your-custom-objects) but with Pytorch.

Thank you!

**Comment by lupotto** [comment]
Hello,
I realised how to make it work with my own dataset.  I modifed the .cfg file, the .names file and .data and it worked. I followed the instructions of the repo that I mentioned above.

**Comment by 0merjavaid** [comment]
@lupotto  Were you able to get results during prediction?
if yes can you please share what changes you have made to get it working?
#34 I am facing this problem.
would really appreciate your help

**Comment by 0merjavaid** [comment]
Resolved by uncommenting this line
From here
model = Darknet(opt.model_config_path)
#model.load_weights(opt.weights_path)
model.apply(weights_init_normal)

To here 

model = Darknet(opt.model_config_path)
model.load_weights(opt.weights_path)
#model.apply(weights_init_normal)

**Comment by lupotto** [comment]
Hi @0merjavaid,

I tried to to make it work with my own dataset/labels, but I thought that the training went smoothly and when I tried to do the testing & prediction didn't work (mAP 0 & 0 detections). Also I tried to retrain with coco dataset doing the same process and happened the same.

Did you resolve the issue changing this line of train.py? After that you loaded the new weights of your new objects/datasets and worked?

Thank you!
